### ISSUE TITLE
Unable to Flag Users from Mod Center

### ISSUE THREAD

**Issue by michael-tharrington** [forem/forem#18458]
_NOTE: there's temporary workaround for this issue. For now, while we cannot access this feature from the mod center, I suggest leaving the mod center and going directly to a post's page or user profile to flag a user._

**Describe the bug**

Both admin and trusted users (and likely tag mods) are unable to flag users from the mod center.

**To Reproduce**

1. Have mod credentials of some form on DEV (or a test Forem)
2. Navigate to https://dev.to/mod
3. Unfurl a post in the list by clicking on its title
4. Look to the tab on the righthand side of the post
5. Look under the heading "Other Things You Can Do"
6. Click the dropdown for moderating actions
7. Click Flag username
8. Notice the lack of confirmation + inability to unflag the user.

**Expected behavior**

Clicking flag username should indeed flag the user. It should put up a warning and ask if you're sure and when you confirm, you should see a confirmation message. It should also change the option from flag user, to unflag user. 

Basically, it needs to mirror the experience of flagging a user while using the mod panel while on a post.

**Screenshots**

https://www.loom.com/share/58e183914ac64e659d91b149199b4935

**Additional context**

A really awesome trusted user, @TanishqSingla, pointed this out to me! 🙌

Also, as an admin, I can't take any of the other actions under "Other Things You Can Do"

**Comment by mirie** [comment]
@PhilipHow I think you're planning on some mod center refreshes -- however, what do you think about adding this one to the Hacktoberfest category of potential issues?

**Comment by michael-tharrington** [comment]
@TanishqSingla just to note that this one works now! Thanks for the heads up. Forgot to close this one down. 🙌
